ingest/rules/nextclade.smk

```diff
@@ -30,7 +30,7 @@ rule run_nextclade:
         # The lambda is used to deactivate automatic wildcard expansion.
         # https://github.com/snakemake/snakemake/blob/384d0066c512b0429719085f2cf886fdb97fd80a/snakemake/rules.py#L997-L1000
         translations=lambda w: "results/translations/{cds}.fasta",
-    threads: 4
+    threads: workflow.cores
```

Comment:
This shaved off 8 mins for the Nextclade job, but only 3 mins off the total workflow. 

Using _all_ of the cores prevents Snakemake from scheduling other concurrent jobs such as `upload_to_s3` jobs. The upload jobs can be slow as well, so we could adjust the threads to something like `workflow.cores * 0.75`. 

It's only a couple minute difference so I'm not going to worry about it for now.